### ISSUE TITLE
increase code coverage by adding unit tests

### ISSUE THREAD

**Issue by mrturkmen06** [pass-wall/passwall-server#116]
**Is your feature request related to a problem? Please describe.**

Not feature but requirement, it is clear that coverage level of the repo could be improved by increasing number of unit tests. It can provide more flexibility and easy debugging from code base without requirement of running server from scratch. 

**Describe the solution you'd like**

- Add more tests where possible 

**Describe alternatives you've considered**

- No alternative solution at the moment.

**Comment by yakuter** [comment]
Yes, we definitely need test which can be small for simple function or big for an endpoint behaviour. Either way, we need tests.

How do we proceed on this subject, where should we start? Do you have suggestions @mrturkmen06 ?

**Comment by mrturkmen06** [comment]
I think, the good starting point could be AUTH and API because those are the crucial components for the project, afterwards database testing, in my opinion.

**Comment by yakuter** [comment]
No problem, this is not a one time job. I will also write test when I find time. Auth and API's are good points to start. Let's keep this issue open to remember every time.
